src/Symfony/Bundle/FrameworkBundle/Resources/config/error_renderer.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
```

Comment:
that seems irrelevant because public=false is the default in sf 4

Reply:
fixed in https://github.com/symfony/symfony/pull/32227, thanks!